apps/src/p5lab/poetry/PoetryLibrary.js

```diff
@@ -26,7 +26,8 @@ export default class PoetryLibrary extends CoreLibrary {
         font: 'Arial'
       },
       isVisible: true,
-      textEffects: []
+      textEffects: [],
+      animationStartFrame: 1
```

Comment:
I think this could benefit from a comment explaining the animationStartFrame - I could see this being confusing without the context of the explanation in the PR.